### ISSUE TITLE
All keyboard-focusable descendants of popup menus should have role menuitem, and menu itself should indicate aria-expanded

### ISSUE THREAD

**Issue by danielweck** [readium/readium-desktop#599]

**Comment by danielweck** [comment]
Fixed via https://github.com/readium/readium-desktop/commit/ca9082a8c32ee867b067782f324a9a09a116e93f
